### PR TITLE
FECK-1905: Allow swig lint accept a public folder param

### PR DESCRIPTION
**Pull request by njprrogers**
- Or configured through package.json

### CODE REVIEW

File: packages/swig-lint/index.js [hunk 1]
```diff
@@ -56,16 +56,24 @@ module.exports = function lintersSetup(gulp, swig) {
...
-              .replace(/\{type\}/g, type)
-              .replace(/\{extension\}/g, extension);
+        .replace(/\{type\}/g, type)
+        .replace(/\{extension\}/g, extension);
     }
 
     if (swig.project.type === 'webapp') {
       baseName = path.basename(swig.target.path);
-      baseSource = path.join(swig.target.path, 'public/{type}/', baseName, '/src/**/*.{extension}');
+      if (swig.argv.public) {
```
Comment (Meesayen) [inline packages/swig-lint/index.js]: `public` -> `src`

I think make more sense, one may want to put his source files in a different folder.

File: packages/swig-lint/index.js [hunk 2]
```diff
@@ -56,16 +56,24 @@ module.exports = function lintersSetup(gulp, swig) {
...
+        .replace(/\{type\}/g, type)
+        .replace(/\{extension\}/g, extension);
     }
 
     if (swig.project.type === 'webapp') {
       baseName = path.basename(swig.target.path);
-      baseSource = path.join(swig.target.path, 'public/{type}/', baseName, '/src/**/*.{extension}');
+      if (swig.argv.src) {
+        publicPath = swig.argv.src;
+      } else if (swig.pkg.gilt && swig.pkg.gilt.publicPath) {
```
Comment (Meesayen) [inline packages/swig-lint/index.js]: `publicPath` -> `srcPath` :D

File: packages/swig-lint/index.js [hunk 3]
```diff
@@ -56,16 +56,24 @@ module.exports = function lintersSetup(gulp, swig) {
   gulp.task('lint-setup', (done) => {
     let baseName;
     let baseSource;
+    let publicPath;
```
Comment (Meesayen) [inline packages/swig-lint/index.js]: `srcPath`

File: packages/swig-lint/index.js (hunk 2 above)
Comment (njprrogers) [inline packages/swig-lint/index.js]: Having a srcPath and a publicPath in the same repo with the same values 99% of the time?

Comment (Meesayen) [inline packages/swig-lint/index.js]: tech-debt.

**Pull request merged**